src/locales/en/messages.json

```diff
@@ -226,11 +226,11 @@
 
   "_load_temporary_plugin": "Load temporary plugin",
 
-  "_reload_theme": "Reload Theme",
+  "_reload_theme": "Reload theme",
```

Comment:
I see you adding new strings capitalised, but vocabulary should be unified.
And since this way is more common, and is better, I am making such changes. 

Please feel free to speak if you want to leave them as they are.